compiler-core/src/error.rs

```diff
@@ -1030,27 +1036,38 @@ https://gleam.run/getting-started/installing/",
                     "rebar3" => text.push_str(
                         "
 Documentation for installing rebar3 can be viewed here:
-https://gleam.run/getting-started/installing/",
-                    ),
-                    "bun" => text.push_str(
-                        "
-Documentation for installing bun can be viewed here:
-https://bun.sh/docs/installation/
-You may need to restart your shell after installing bun.",
+https://gleam.run/getting-started/installing/
```

Comment:
Let's have just the official ones and remove the Gleam link please

Reply:
i think there wasn't a gleam link for `bun`